site/ko/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools 업데이트 소식 (Chrome 93)"
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "수정 가능한 CSS 컨테이너 쿼리, 웹 번들 프리뷰, Console 에서 더 나은 문자열 제어 및 추가 업데이트 소식"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cNXHu1JnHCfRSnIr1bV7.jpg'
```

Comment:
Update image since we change the title `image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Cs7C4adVQQjG54wEmBch.jpg`

Reply:
How can I help changing the image? Is there a source file for the image?

Reply:
https://wd.imgix.net/image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Cs7C4adVQQjG54wEmBch.jpg?auto=format&w=1600